pipeline-templates/step/deploy.yml

```diff
@@ -4,9 +4,11 @@ parameters:
   AzureServiceConnection:
 
 steps:
+- checkout: self
+- checkout: das-platform-automation
```

Comment:
It doesn't look like any of the steps use artifacts from das-platform-automation, do we need to check this out?

Reply:
Thanks, that was added when I going to whitelist the hosted agent on the K8 API which testing deemed not suitable. 

Cleaned up now.